packages/tgpu-cli/gen.mjs

```diff
@@ -0,0 +1,25 @@
+import { promises as fs } from 'node:fs';
+
+const cwd = new URL(`file:${process.cwd()}/`);
+
+async function placeholderGenerate(input, output) {
+  const inputPath = new URL(input, cwd);
+  const outputPath = new URL(output, cwd);
+
+  const inputContents = await fs.readFile(inputPath, 'utf8');
+  const placeholderTS = `
+// This file is a placeholder for the actual implementation
+
+export default function placeholder() {
+  const input = \`
+  ${inputContents}
+  \`;
```

Comment:
`JSON.stringify` is a good way to escape any unwanted characters from the input string.

```suggestion
  const input = ${JSON.stringify(inputContents)
        .replace(/\u2028/g, '\\u2028')
        .replace(/\u2029/g, '\\u2029')};
```